app/components/provider_interface/dsa_success_page_component.html.erb

```diff
@@ -0,0 +1,77 @@
+<% if FeatureFlag.active?(:accredited_provider_setting_permissions) %>
```

Comment:
It's now much more explicit what the feature flag is doing, and it's easy to delete